python_modules/libraries/dagster-dg/dagster_dg/cli/plus/deploy.py

```diff
@@ -59,13 +59,13 @@ def _get_deployment(input_deployment: Optional[str], plus_config: DagsterPlusCli
                 ["--organization"],
                 "organization",
                 help="Dagster+ organization to which to deploy. If not set, defaults to the value set by `dg plus login`.",
-                envvar="DAGSTER_PLUS_ORGANIZATION",
+                envvar="DAGSTER_CLOUD_ORGANIZATION",
```

Comment:
I realize this is just changing extant code, I do wish we had some more explicit telegraphing of what configuration we're using (mainly if the user has env var and config file set, we loudly state that the override is happening)

Reply:
I can add a follow-up to do this